lib/log_blackbox.c

```diff
@@ -188,7 +188,7 @@ qb_log_blackbox_write_to_file(const char *filename)
 	ssize_t written_size = 0;
 	struct qb_log_target *t;
 	struct _blackbox_file_header header;
-	int fd = open(filename, O_CREAT | O_RDWR, 0700);
+	int fd = open(filename, O_CREAT | O_RDWR | O_EXCL, 0700);
```

Comment:
`O_RDWR` shall likely be flipped to `O_WRONLY`.

`O_EXCL` is dubious, overwriting existing file is a valid use case
here and an established semantics of the function.

Perhaps devising `qb_log_blackbox_write_to_file_2` with some expressly
noted flags passable as `flags` and making `qb_log_blackbox_write_to_file`
depend on that might be a better idea.